Round robin edits - AH (#247)
AlexsLemonade/ScPCA-manuscript: content/03.results.md

The Portal also provides for visualization of individual samples via the UCSC Cell Browser interface [@doi:10.1093/bioinformatics/btab503] as seen in Figure {@fig:fig1}C.
Interactive UMAPs allow users to explore the cells within each sample, coloring by cell type annotations, gene expression values, or other calculated metrics.

### Uniform processing of data available on the ScPCA Portal

We developed [`scpca-nf`](https://github.com/AlexsLemonade/scpca-nf), an open-source and efficient Nextflow [@doi:10.1038/nbt.3820] workflow for quantifying single-cell and single-nuclei RNA-seq data and processed all data available on the Portal with it.
Nextflow is a workflow management system that allows users to execute multi-step and long-running bioinformatics processes in a portable and reproducible manner across various computing environments, including high-performance computing clusters and cloud-based computing [@doi:10.1186/s13059-025-03673-9].

> Nextflow minimizes run time by parallelizing sample processing as allowed by the environment, and automatically manages all dependencies, as each process in the workflow is run in a Docker or Singularity container.

Author: ```suggestion
Nextflow allows seamless management of all dependencies, as each process in the workflow can be run using a specified container image. 
```
Just a slight rewording of what you suggested.

The combination of being able to execute a Nextflow workflow in any environment and run individual processes in Docker containers makes this workflow easily portable for external use.

> Setup requires only installing Nextflow and a supported container engine like Docker or Apptainer, managing a single configuration file for the computing environment, and organizing input files.

Reviewer: Since `Apptainer` is not in the dictionary, maybe we don't need either?
```suggestion
Setup requires only installing Nextflow and a supported container engine, managing a single configuration file for the computing environment, and organizing input files.
```

An example of this is shown in Figure {@fig:figS4}A, which displays cells that are annotated as different T cell types by each automated method.
T cell annotations are then harmonized across all methods by assigning the LCA with the fewest descendants as the consensus cell type, providing a single, consistent label for each cell (Figure {@fig:figS4}B).

We validated consensus cell types by evaluating cell-type-specific marker gene expression across all cells (Figure {@fig:fig4}A, Figure {@fig:figS5}), observing high concordance between consensus cell type labels and expected marker gene expression.
Note that a library-specific version of these marker gene expression dot plots is provided in the summary QC report.
In addition, an expanded version of the heatmap shown in Figure {@fig:fig3}C with all consensus cell types is provided in the summary cell type report.
These visualizations allow users to assess the reliability of consensus cell type annotations and clearly see how cell type labels from different automated methods contribute to the final consensus label.

> #### Consensus cell type annotations in brain and CNS samples available on the Portal

Reviewer: That makes sense.

We also see traces of this relationship even when looking at the consensus cell types in conjunction with CNV events.
Figure {@fig:fig5}D shows the distributions of per-cell total CNV events for the most commonly-observed consensus cell types in the neuroblastoma library `SCPCL000130`.
Here, unknown and neuron cells have distinctly higher total CNV values compared to other cell types, suggesting that they are likely malignant.
We see similar patterns for the ganglioglioma library `SCPCL000049` (Figure {@fig:figS4}B-C), where consensus immune cell types have low total CNV values, while other cell types including oligodendrocyte precursor cells, neuron associated cells, and unknown cells have much higher total CNV values.
As such, joint information from consensus cell type annotations and `InferCNV` results can help identify malignant cells across libraries in the Portal, including those which do not yet have associated OpenScPCA project annotations.

### Analysis of bulk RNA-seq

> Several projects in the ScPCA Portal contain bulk RNA-seq data in addition to single-cell or single-nuclei RNA-seq data.

Reviewer: we may want to revert these (or change all) for word count later.

Previous research has suggested that, compared to bulk RNA-seq, single-cell and single-nuclei RNA-seq technologies may fail to capture certain cell types [@doi:10.1038/s41587-020-0465-8], for example, due to technical aspects of library preparation.
We therefore asked whether we could identify differences in biological signal between these two modalities that may suggest distinct cell type distributions.
We specifically focused on ScPCA projects with solid tumors, considering only samples with both sequencing modalities, and excluded low-quality single-cell or single-nuclei libraries and multiplexed samples.
We analyzed 97 samples across five projects: `SCPCP00001` (high-grade glioma), `SCPCP000002` (low-grade glioma), `SCPCP000006` (Wilms tumor), `SCPCP000009` (CNS tumors), and `SCPCP000017` (osteosarcoma).

> As described in the Methods, we derived pseudobulk expression matrices for each single-cell or single-nuclei library, and we compared their expression to bulk using a series of linear models (one per ScPCA project) predicting bulk from pseudobulk expression with a random effect controlling for sample (Figure {@fig:fig6}A, Figure {@fig:figS7}A).

Reviewer: ```suggestion
As described in the Methods, we derived pseudobulk expression matrices for each single-cell or single-nuclei library and compared the pseudobulk expression to bulk using a series of linear models (one per ScPCA project) with a random effect controlling for sample (Figure {@fig:fig6}A, Figure {@fig:figS7}A).
```
